classifiers/python_functions/profanity_detection/code_snippet.md

```diff
@@ -1,9 +1,11 @@
 ```python
 from better_profanity import profanity
 
-YOUR_ATTRIBUTE = "body"
+YOUR_ATTRIBUTE: str = "text"
+YOUR_LABEL_01: str = "profane"
```

Comment:
maybe YOUR_LABEL_PROFANE and YOUR_LABEL_NOT_PROFANE to have a better name for the enduser